tests/filecheck/dialects/omp/ops.mlir

```diff
@@ -224,6 +224,22 @@ builtin.module {
     }) : () -> ()
     func.return
   }
+  "omp.private"() <{data_sharing_type = #omp.data_sharing_type {type = private}, sym_name = "p1", type = i32}> ({
```

Comment:
Could you please add the assembly format for these functions?

Reply:
I might have time today to implement this.

Reply:
https://github.com/xdslproject/xdsl/pull/4811 I promise.